src/oscar/templates/oscar/dashboard/catalogue/option/form.html

```diff
@@ -0,0 +1,83 @@
+{% extends 'dashboard/layout.html' %}
```

Comment:
To be consistent with all the other templates I'd suggest renaming this to `dashboard/option_form.html`. Similarly `option_list.html`, `option_row_actions.html`.

Reply:
Maybe it should be in discuss list? Because currently managing all catalog templates in single folder is difficult.